compose/install.md

```diff
@@ -34,28 +34,59 @@ To install Compose, do the following:
     Invoke-WebRequest "https://github.com/docker/compose/releases/download/$dockerComposeVersion/docker-compose-Windows-x86_64.exe" -UseBasicParsing -OutFile $Env:ProgramFiles\docker\docker-compose.exe
     ```
 
-    For example, to download Compose version 1.12.0, the command is:
+    For example, to download Compose version 1.13.0, the command is:
 
     ```none
-    Invoke-WebRequest "https://github.com/docker/compose/releases/download/1.12.0/docker-compose-Windows-x86_64.exe" -UseBasicParsing -OutFile $Env:ProgramFiles\docker\docker-compose.exe
-        ```
+    Invoke-WebRequest "https://github.com/docker/compose/releases/download/1.13.0/docker-compose-Windows-x86_64.exe" -UseBasicParsing -OutFile $Env:ProgramFiles\docker\docker-compose.exe
+    ```
+    >**Use the latest Compose release number in the download command.**
+    As already mentioned, the above command is an _example_, and it may become
+out-of-date once in a while. Always follow the command pattern shown above it.
```

Comment:
You should line up lines 44+ with line 42. It's also safer to use the `>` at the beginning of each line in the note.